docker_custodian/docker_gc.py

```diff
@@ -46,16 +48,21 @@ def is_container_old(container, min_date):
     if state.get('Ghost'):
         return True
 
+    created_date = dateutil.parser.parse(container['Created'])
+
+    # Don't delete recently created containers
+    if max_date and created_date > max_date:
+        return False
+
     # Container was created, but never used
-    if (state.get('FinishedAt') == YEAR_ZERO and
-            dateutil.parser.parse(container['Created']) < min_date):
+    if state.get('FinishedAt') == YEAR_ZERO and created_date < min_date:
```

Comment:
I agree using the YEAR_ZERO is fragile, unfortunately I haven't found a better way to identify containers in this state (created but not started).

It would be great to have some idea about why this is not working correctly.